web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -37,7 +38,12 @@
       <div
         class="absolute flex h-16 w-full place-items-center justify-between border-b p-4 dark:border-immich-dark-gray dark:text-immich-dark-fg"
       >
-        <p class="font-medium">{title}</p>
+        <div class="flex gap-2 items-center">
+          <div class="font-medium">{title}</div>
+          {#if secondTitle}
```

Comment:
Let's rename this to `description` or something like that, especially if it is small text in a p tag.